is_prime.py

```diff
@@ -0,0 +1,9 @@
+def is_prime(n):
```

Comment:
I would highly recommend getting in a habit of adding docstrings to your functions. You can technically use any info with `"""` , but if you add a function description along with input/outputs in some standard format, you can leverage auto-documentation tools to make working with your code easier.

https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings